cfpb/consumerfinance.gov
# Adding categories for petitions

This PR adds two new categories within the new topic of "Petitions", specifically to add categories as additional information for Petitions for Rulemaking child pages (`/rules-policy/petitions-rulemaking/`).

---




## Additions

- Category options in `ref.py`



## How to test this PR

1. localhost
2. Add categories to any child pages under `/rules-policy/petitions-rulemaking/`
3. On `/rules-policy/petitions-rulemaking/`, make sure the Filtering option for categories is checked


## Screenshots
Current view of an option on `/rules-policy/petitions-rulemaking/`:
![Screenshot 2023-08-09 at 7 14 48 AM](https://github.com/cfpb/consumerfinance.gov/assets/29648039/53d37dde-6a97-457a-ace1-28f8a3e095c7)

Option on localhost with the category set:
![Screenshot 2023-08-09 at 7 14 39 AM](https://github.com/cfpb/consumerfinance.gov/assets/29648039/bef0d302-a253-4517-b6b4-a5c23074787c)


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
